converters/google/resources/cai.go

```diff
@@ -99,10 +99,20 @@ type RestoreDefault struct {
 func assetName(d TerraformResourceData, config *Config, linkTmpl string) (string, error) {
 	re := regexp.MustCompile("{{([[:word:]]+)}}")
 
+	// workaround for empty project
+	placeholderSet := false
+	if config.Project == "" {
+		config.Project = "placeholder-project"
```

Comment:
I think it might be better to have a random placeholder here to match the existing logic. The test suite should automatically ignore comparison of placeholders.

```suggestion
		config.Project = fmt.Sprintf("placeholder-%s", randString(8))
```

Would there be any issue doing it that way?

Reply:
Done, had to adjust init_test.go to normalize the fields in the resource data to make that work.

Reply:
ah, gotcha. That seems like it'll be okay 👍